src/constraints/storage_constraints.jl

```diff
@@ -207,6 +220,17 @@ function add_cold_thermal_storage_dispatch_constraints(m, p, b; _n="")
         m[Symbol("dvStoragePower"*_n)][b] >= m[Symbol("dvDischargeFromStorage"*_n)][b,ts] + 
         sum(m[Symbol("dvProductionToStorage"*_n)][b,t,ts] for t in p.techs.cooling)
     )
+
+    #Do not allow GHP to charge storage
+    if !isempty(p.techs.ghp)
+        for b in p.s.storage.types.cold
```

Comment:
Could this be written as 

`for b in p.s.storage.types.cold, t in p.techs.ghp, ts in p.time_steps` ?

Reply:
yes, that's a better way!  I implemented it in 2228b394f823e8c8ce2f76c3ef52477054bac302